src/source.rs

```diff
@@ -4,9 +4,9 @@
 
 use std::cell::RefCell;
 use std::mem::transmute;
-#[cfg(any(feature = "v2_36", unix))]
+#[cfg(all(unix, any(feature = "v2_36", feature = "dox")))]
 use std::os::unix::io::RawFd;
-#[cfg(any(feature = "v2_36", windows))]
+#[cfg(all(windows, any(feature = "v2_36", feature = "dox")))]
 use libc::c_int as RawFd;
```

Comment:
Now it don't build docs for other that unix/windows.
Maybe use `cfg(all(not(unix), feature = "dox"))`, it works for me.

Reply:
Ok, defaulting to libc::c_int should be fine, but does glib actually support anything else?

Reply:
IMHO if no `libc::c_int` then platform don't have support at all.